client/src/components/layout/Navbar.js

```diff
@@ -94,7 +114,7 @@ const Navbar = ({ auth: { isAuthenticated, loading, user }, logout }) => {
               <li onClick={(e) => handleClick(e)}>
                 <Avatar
                   alt={user.name}
-                  src='/static/images/avatar/2.jpg'
+                  src={`https://res.cloudinary.com/daqdhcvyv/image/upload/v1615793443/${image}`}
```

Comment:
hm this entire cdn, instead of just storing ${image} it would be better to store the entire url